CODEOWNERS

```diff
@@ -30,6 +30,7 @@
 /anda/audio/distrho @korewaChino
 
 /anda/desktop/compiz9 @cat-master21
+/anda/lib/glewmx @cat-master21
 /anda/lib/libindicator @cat-master21
 /anda/lib/libunity @cat-master21
 /anda/lib/libunity-misc-devel @cat-master21
```

Comment:
oh can you remove this? It doesn't really work for people outside of the team members

we'll update the wiki